test/wasm.js

```diff
@@ -8,7 +8,41 @@ function puts(p) {
   print(s);
 }
 
-ffi = {print: print, puts: puts};
+function writeByte(b) {
+  var s = b.toString();
+  while (s.length < 3) {
+    s = " " + s;
+  }
+  write(s);
+}
+
+function flipBuffer(p, w, h) {
+  print("P3");
+  print(w + " " + h);
+  print("255");
+  print("# This is a PPM file, redirect stdout to view.");
```

Comment:
1) it wasn't clear how to output binary data from d8.  I heard through the grapevine today that you hacked something together?
2) easier to diff if we turn it into a test.
3) it doesn't cause as much damage if it gets dumped to the terminal.  Meh.

But really, the main motivation was 1).